### DISCUSSION TITLE
GLSP figure size increasing while showing the node name.

### DISCUSSION THREAD

**Discussion by harshad-mohape** [eclipse-glsp/glsp#1139] [Q&A]
Hello,
I am currently working on a graph and I need to show the label of the component.

In my current scenario, I have created a Circular shape and added the label as a children to that node. However, the size of the figure is increasing if size of label text is bigger.

thanks in advance.

**Comment by CamilleLetavernier** [comment]
Hi,

Your Circle node probably has default layout options, which include `resizeContainer=true`. This indicates that if children are larger than the parent, then the parent should be automatically resized to match its children size. Setting this layout option to false instead (on your Circle node), might solve your issue.

Hope this helps,
